installer/templates/phx_assets/webpack/package.json

```diff
@@ -17,7 +17,7 @@
     "css-loader": "^0.28.10",
     "mini-css-extract-plugin": "^0.4.0",
     "optimize-css-assets-webpack-plugin": "^4.0.0",
-    "uglifyjs-webpack-plugin": "^1.2.4",
+		"terser-webpack-plugin": "^1.1.0",
```

Comment:
Can you use spaces instead of tabs here please?

Reply:
Done.